aspnet/web-forms/overview/deployment/visual-studio-web-deployment/deploying-to-production.md

```diff
@@ -37,7 +37,7 @@ If you don't already have an Azure account, you can create a free trial account
 ## Create a staging environment
 
 > [!NOTE]
-> Since this tutorial was written, Azure App Service added a new feature to automate many of the processes around having staging and production environments. See [Set up staging environments for web apps in Azure App Service](https://azure.microsoft.com/en-us/documentation/articles/web-sites-staged-publishing/).
+> Since this tutorial was written, Azure App Service added a new feature to automate many of the processes around having staging and production environments. See [Set up staging environments for web apps in Azure App Service](https://azure.microsoft.com/documentation/articles/web-sites-staged-publishing/).
```

Comment:
The words "around having" in the first sentence are a bit vague. Should this say "for setting up" instead?